packages/webview_flutter/webview_flutter_android/lib/src/v4/src/android_webview_controller.dart

```diff
@@ -266,14 +303,79 @@ class AndroidWebViewController extends PlatformWebViewController {
+  /// Sets whether the WebView should use its built-in zoom mechanisms.
+  ///
+  /// The built-in zoom mechanisms comprise on-screen zoom controls, which are
+  /// displayed over the WebView's content, and the use of a pinch gesture to
+  /// control zooming. Whether or not these on-screen controls are displayed can
+  /// be set with [setDisplayZoomControls]. The default is false.
+  ///
+  /// The built-in mechanisms are the only currently supported zoom mechanisms,
+  /// so it is recommended that this setting is always enabled. However,
+  /// on-screen zoom controls are deprecated in Android so it's recommended to
+  /// disable [setDisplayZoomControls].
+  Future<void> setDisplayZoomControls(bool enabled) =>
+      _webView.settings.setDisplayZoomControls(enabled);
+
+  /// Sets whether the WebView loads pages in overview mode, that is, zooms out the content to fit on screen by width.
+  ///
+  /// This setting is taken into account when the content width is greater than
+  /// the width of the WebView control, for example, when [setUseWideViewPort]
+  /// is enabled.
+  ///
+  /// The default is false.
+  Future<void> setLoadWithOverviewMode(bool enabled) =>
+      _webView.settings.setLoadWithOverviewMode(enabled);
+
   @override
   Future<void> setJavaScriptMode(JavaScriptMode javaScriptMode) =>
       _webView.settings
           .setJavaScriptEnabled(javaScriptMode == JavaScriptMode.unrestricted);
 
+  /// Sets wether JavaScript is allowed to open windows automatically.
+  ///
+  /// This applies to the JavaScript function `window.open()`. The default is
+  /// false.
+  Future<void> setJavaScriptCanOpenWindowsAutomatically(bool enabled) =>
+      _webView.settings.setJavaScriptCanOpenWindowsAutomatically(enabled);
+
+  // TODO(bparrishMines): Update documentation when WebChromeClient.onCreateWindow is added.
+  /// Sets whether the WebView should supports multiple windows.
+  ///
+  /// The default is false.
+  Future<void> setSupportMultipleWindows(bool support) =>
+      _webView.settings.setSupportMultipleWindows(support);
+
+  /// Sets whether the WebView should enable support for the "viewport" HTML meta tag or should use a wide viewport.
+  ///
+  /// When the value of the setting is false, the layout width is always set to
+  /// the width of the WebView control in device-independent (CSS) pixels. When
+  /// the value is true and the page contains the viewport meta tag, the value
+  /// of the width specified in the tag is used. If the page does not contain
+  /// the tag or does not provide a width, then a wide viewport will be used.
+  Future<void> setUseWideViewPort(bool enabled) =>
+      _webView.settings.setUseWideViewPort(enabled);
+
   @override
   Future<void> setUserAgent(String? userAgent) =>
       _webView.settings.setUserAgentString(userAgent);
+
+  /// Sets the restrictions that apply on automatic media playback.
+  Future<void> setAutoMediaPlaybackPolicy(
```

Comment:
Since this is Android only, I think this would be better as `setMediaPlaybackRequiresUserGesture(bool)`. The enum was used before because iOS could set the specific media that requires gestures.

Reply:
Updated the method callback to match the Android specific name.